conductr_cli/resolvers/bintray_resolver.py

```diff
@@ -32,7 +32,7 @@ def resolve_bundle(cache_dir, uri):
 def load_bundle_from_cache(cache_dir, uri):
     # When the supplied uri points to a local file, don't load from cache so file can be used as is.
     parsed = urlparse(uri, scheme='file')
-    if parsed.scheme == 'file' and os.path.exists(uri):
+    if parsed.scheme == 'file' and parsed.path.endswith('.zip') and os.path.exists(uri):
```

Comment:
This also needs to be applied to `load_bundle_configuration_from_cache`:

https://github.com/typesafehub/conductr-cli/pull/226/files#diff-fd50a12e0d19d40e43b9178ed70df60fR68